ballerina/types.bal

```diff
@@ -259,3 +259,587 @@ public enum LogLevel {
     @display {label: "OFF"}
     OFF
 }
+
+# SQL Rule
+#
+# + filter - Represents a filter which is a composition of an expression and an action that is executed in the pub/sub pipeline.
```

Comment:
```suggestion
# + filter - Represents a filter which is a composition of an expression and an action that is executed in the pub/sub pipeline
```

Reply:
Addressed in [144f075](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/180/commits/144f075302b1178fb900ff7bfee769bc987f6759)